api/src/main/java/io/strimzi/api/kafka/model/balancing/KafkaRebalanceState.java

```diff
@@ -21,12 +21,12 @@ public enum KafkaRebalanceState {
      * A proposal has been requested from Cruise Control, but is not ready yet.
      * Transitions to:
      * <dl>
-     *     <dt>PendingProposal</dt><dd>A rebalance proposal is not ready yet.</dd>
+     *     <dt>PendingProposal</dt><dd>A rebalance proposal is not ready yet or strimzi.io/rebalance=refresh is applied and a new proposal is requested.</dd>
```

Comment:
Do we need to specify `or strimzi.io/rebalance=refresh is applied and a new proposal is requested` ? I mean, this is the description of the state and it is actually `A rebalance proposal is not ready yet` ... now if it happens because you create the KafkaRebalance or refresh is not part of the state itself.

Reply:
Yeah, I will fix this